frontend/src/hooks/useResourceAttribute/utils.ts

```diff
@@ -93,6 +93,22 @@ export const resourceAttributesToTagFilterItems = (
 		value: `${res.tagValue}`.split(','),
 	}));
 };
+/* Convert resource attributes to trace filters items for queryBuilder */
+export const resourceAttributesToTracesFilterItems = (
+	queries: IResourceAttribute[],
+): TagFilterItem[] =>
+	queries.map((res) => ({
+		id: `${res.id}`,
+		key: {
+			key: convertMetricKeyToTrace(res.tagKey),
+			isColumn: false,
+			type: MetricsType.Resource,
+			dataType: DataTypes.String,
+			id: `${convertMetricKeyToTrace(res.tagKey)}--string--resource--true`,
```

Comment:
better to also spread `res` at the end to not miss any current and future key-value like `isJSON`

Reply:
This is not required in the request payload. Keep it if it's required on the frontend otherwise, remove it.

Reply:
@SagarRajput-7, here we get res in the following format:
```
{
    "id": "1cc16351",
    "tagKey": "resource_host_name",
    "operator": "Not IN",
    "tagValue": [
        "abc3421ab"
    ]
}
```

I guess we don't need to spread it. Thoughts?